homeassistant/components/nordpool/quality_scale.yaml

```diff
@@ -0,0 +1,101 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry:
+    status: exempt
+    comment: |
+      This integration only allows single config entry through manifest
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities doesn't subscribe to events.
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: todo
+  docs-removal-instructions: todo
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  brands: done
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: |
+      No actions.
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      This integration does not require authentication.
+  parallel-updates: todo
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: todo
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      No options to configure
+
+  # Gold
+  entity-translations: done
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default: done
+  discovery:
+    status: exempt
+    comment: |
+      No discovery, cloud service
+  stale-devices:
+    status: exempt
+    comment: |
+      This integration devices (services) will be removed with config entry if needed.
+  diagnostics: done
+  exception-translations:
```

Comment:
`done` you have at least one in init. Why are you never raising `UpdateFailed` btw (e.g. after consecutive failures)?

Reply:
It has it's own update mechanism so therefore I'm setting the variable myself instead of using the built-in functionality (which triggers by raising `UpdateFailed`)